telemetry/packages/ui/app/context/pods.tsx

```diff
@@ -44,6 +46,10 @@ const LATENCY_REQUEST_INTERVAL = 100 as const;
  * The default pod ID to use
  */
 export const DEFAULT_POD_ID = POD_IDS[1];
+/**
+ * The default mode to run
+ */
+export const DEFAULT_MODE = MODES.ALL_SYSTEMS_ON as ModeType;
```

Comment:
Once the config is in place we don't want to have a default, we'll just look for the value in the config and throw an error if it's not there.